src/current/v23.1/molt-verify.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Use the MOLT Verify tool
+summary: Learn how to use the MOLT Verify tool to check for post-migration data discrepancies.
+toc: true
+docs_area: migrate
+---
+
+The MOLT Verify tool checks for post-migration data discrepancies between a source and a target database.
+
+The tool performs the following verifications to ensure post-migration data integrity:
```

Comment:
Conceptually, I believe data verification is a prerequisite to a successful (i.e., consistent) migration. So I think we should replace "post-migration" with phrasing like "during a migration", "as part of a migration", etc.

Reply:
@awoods187 @cotedm Please tell me if I have this wrong!

Reply:
I agree--well spotted